docs/api-inference/index.md

```diff
@@ -8,14 +8,14 @@ Explore the most popular models for text, image, speech, and more — all with a
 
 ## Why use the Inference API?
 
-The Serverless Inference API offers a fast and free way to explore thousands of models for a variety of tasks. Whether you're prototyping a new application or experimenting with ML capabilities, this API gives you instant access to high-performing models across multiple domains:
+The Serverless Inference API offers a fast and simple way to explore thousands of models for a variety of tasks. Whether you're prototyping a new application or experimenting with ML capabilities, this API gives you instant access to high-performing models across multiple domains:
 
 * **Text Generation:** Including large language models and tool-calling prompts, generate and experiment with high-quality responses.
 * **Image Generation:** Easily create customized images, including LoRAs for your own styles.
 * **Document Embeddings:** Build search and retrieval systems with SOTA embeddings.
 * **Classical AI Tasks:** Ready-to-use models for text classification, image classification, speech recognition, and more.
 
-⚡ **Fast and Free to Get Started**: The Inference API is free with higher rate limits for PRO users. For production needs, explore [Inference Endpoints](https://ui.endpoints.huggingface.co/) for dedicated resources, autoscaling, advanced security features, and more.
+⚡ **Fast and Free to Get Started**: The Inference API is free to try out and offers additional included credits for PRO users. For production needs, explore [Inference Endpoints](https://ui.endpoints.huggingface.co/) for dedicated resources, autoscaling, advanced security features, and more.
```

Comment:
```suggestion
⚡ **Fast and Free to Get Started**: The Inference API is free to try out and comes with additional included credits for PRO users. For production needs, explore [Inference Endpoints](https://ui.endpoints.huggingface.co/) for dedicated resources, autoscaling, advanced security features, and more.
```

not sure tbh - it read a bit weird (feel free to ignore)

Reply:
better than mine, i'll merge it